init.el

```diff
@@ -13,6 +13,54 @@
   :link '(url-link "https://github.com/SystemCrafters/rational-emacs")
   :group 'emacs)
 
+;;; package configuration
+(defun rational-package-archives-stale-p ()
+  "Check on-disk archives last modified date to see if it is stale.
+
+Return non-nil if the on-disk cache is older than one day or
+`nil' otherwise."
+  (let ((today (time-to-days (current-time)))
+        result)
+   (dolist (archive package-archives result)
+     (let* ((archive-name (expand-file-name (format "archives/%s/archive-contents" archive)
+                                            package-user-dir))
+            (archive-modified-date (time-to-days
+                                    (file-attribute-modification-time
+                                     (file-attributes archive-name)))))
+       (when (time-less-p archive-modified-date today)
+         (setq result t))))))
+
+(defmacro rational-install-package (package)
```

Comment:
Should this be `rational-package-install-package`? Seems a bit redundant, but would match the naming conventions.

Reply:
I can fix that... but not necessary I get rid of the macro. (see #94 response)